test/built-ins/RegExp/dotall/with-dotall.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Without the dotAll flag, . does not match newlines
+info: |
+  21.2.2.8 Atom
+  The production Atom::. evaluates as follows:
+    1. If DotAll is true, then
+      a. Let A be the set of all characters.
+    2. Otherwise, let A be the set of all characters except LineTerminator.
+    3. Call CharacterSetMatcher(A, false) and return its Matcher result.
+
+esid: sec-atom
+features: [regexp-dotall]
+---*/
+
+// DotAll '.' behavior.
+{
+  let re = /^.$/s;
+  assert.sameValue(true, re.test("a"));
```

Comment:
this is weird. `true` should be the expected value as `assert.sameValue(actual, expected);` This format breaks convention.